dttools/test/test_runner_common.sh

```diff
@@ -138,7 +138,7 @@ run_taskvine_worker()
 		exit 1
 	fi
 	echo "Running worker."
-	if ! "$TASKVINE_WORKER" --single-shot --timeout=10s --cores ${cores:-1} --memory ${memory:-250} --disk ${disk:-250} --gpus ${gpus:-0} --debug=all --debug-file="$log" $* localhost $(cat "$port_file"); then
+	if ! "$TASKVINE_WORKER" --single-shot --timeout=200s --cores ${cores:-1} --memory ${memory:-250} --disk ${disk:-250} --gpus ${gpus:-0} --debug=all --debug-file="$log" $* localhost $(cat "$port_file"); then
```

Comment:
The "s" in 200s is not needed. It is ignored by `atoi`.
Also, I suggest increasing the timeout value only for particular tests, otherwise failing tests may take very long to complete.  Look at how gpus, memory, etc. are specified from the test and do something similar for something with a name like `idle_timeout`.  The default idle_timeout should be 10.

Reply:
I've changed it.